recipes/example/meta.yaml

```diff
@@ -1,82 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
```

Comment:
please restore the example

Reply:
Thanks. I was confused by the directions, and wasn't sure what the end goal of the PR was.